pkg/dockerfile/generator.go

```diff
@@ -11,11 +11,35 @@ import (
 	"strings"
 
 	"github.com/replicate/cog/pkg/config"
+	"github.com/replicate/cog/pkg/modelfinder"
 )
 
 //go:embed embed/cog.whl
 var cogWheelEmbed []byte
 
+const DockerignoreHeader = `# generated by replicate/cog
+__pycache__
+*.pyc
+*.pyo
+*.pyd
+.Python
+env
+pip-log.txt
+pip-delete-this-directory.txt
+.tox
+.coverage
+.coverage.*
+.cache
+nosetests.xml
+coverage.xml
+*.cover
+*.log
+.git
+.mypy_cache
+.pytest_cache
+.hypothesis
+`
```

Comment:
It'd be helpful to annotate what each of these are:

```suggestion
const DockerignoreHeader = `
# generated by replicate/cog

# Python cache files
__pycache__
*.pyc
*.pyo
*.pyd
.Python

# Virtual environments
env

# Log files
pip-log.txt

# Files generated by pip
pip-delete-this-directory.txt

# Test environments for Python
.tox

# Code coverage tools
.coverage
.coverage.*
.cache
nosetests.xml
coverage.xml
*.cover

# Log files
*.log

# Git directory
.git

# Cache for type check tool
.mypy_cache

# Pytest cache directory
.pytest_cache

# Hypothesis (property-based testing tool) directory
.hypothesis
`
```

Reply:
Another version that groups entries in a slice (requires additional changes to generate `.dockerignore` file):

```suggestion
var DockerignoreEntries = []string{
	"__pycache__, *.pyc, *.pyo, *.pyd, .Python",  // Python cache files and debug info
	"env",  // Virtual environments
	"pip-log.txt, pip-delete-this-directory.txt",  // Pip log files and generated files
	".tox",  // Test environments for Python
	".coverage, .coverage.*, .cache, nosetests.xml, coverage.xml, *.cover",  // Code coverage tools
	"*.log",  // Log files
	".git",  // Git directory
	".mypy_cache",  // Cache for type check tool
	".pytest_cache",  // Pytest cache directory
	".hypothesis",  // Hypothesis (property-based testing tool) directory
}
```